src/validation/matrix-sort-type.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Matrix Sort Type Validation
+ */
+
+module.exports = (arg, func) => {
+    if (arg != 'row' || arg != 'col') {
+        throw new TypeError(`Invalid argument received: ${JSON.stringify(arg)}\n'${func}()' only accept 'row' or 'col'!\n`);
+    };
+}
```

Comment:
It's good to have an empty line at the end of file

Reply:
Ohkay Varun, I'll keep in mind.